frontend_components/src/encord_active_components/frontend/src/components/explorer/Explorer.tsx

```diff
@@ -116,6 +120,7 @@ export const Explorer = ({
   }, [metrics?.length]);
 
   useEffect(() => {
+    if (isLoadingSortedItems) return;
     setSortedAndFiltered(
       sortedItems?.filter(({ id }) => itemSet.has(id)) || []
     );
```

Comment:
nit: isLoadingSortedItems should be added to the dependency list (due to correlation with sortedItems not full needed)

2nd nit: is setSortedAndFiltered another memo in disguise?

Reply:
first nit: Absolutely

2nd nit: not really since we set it from other places as well